Build/build.ps1

```diff
@@ -133,6 +133,10 @@ task Package -depends Build {
   robocopy $sourceDir $workingDir\Package\Source\Src /MIR /NFL /NDL /NJS /NC /NS /NP /XD bin obj TestResults AppPackages .vs artifacts /XF *.suo *.user *.lock.json | Out-Default
   robocopy $buildDir $workingDir\Package\Source\Build /MIR /NFL /NDL /NJS /NC /NS /NP /XD Temp /XF runbuild.txt | Out-Default
   robocopy $docDir $workingDir\Package\Source\Doc /MIR /NFL /NDL /NJS /NC /NS /NP | Out-Default
+  
+  # include fuzz tests in ADO pipeline artifacts
+  mkdir $workingDir\FuzzTests
```

Comment:
also, curious where the pipeline changes are made for enabling fuzzing ? are there any yaml file changes somewhere ?

Reply:
This is defined in a separate pipeline which consumes the output of this build pipeline, it makes it easier if we need to change any configuration settings.